extension/homepage/homepage.scss

```diff
@@ -0,0 +1,226 @@
+@use '../css/vendor/protocol/protocol';
```

Comment:
This is where the error is happening. Maybe you put those files into place locally but it's not part of the build? Or the build might be happening in a different order, so those files haven't been copied in. I suppose this might also be a way to use that sass without copying it at all: https://www.npmjs.com/package/node-sass-package-importer